### DISCUSSION TITLE
Installation logs me out

### DISCUSSION THREAD

**Discussion by huyz** [pop-os/shell#1552] [Q&A] [answered]
`make local-install` seems to kill the display server.  And then I just get a blank screen.

I'm on Ubuntu 22.04.1LTS arm64 running inside Parallels 18 for macOS 12.6.1 (Monterey) Apple M1 Max (ARM).

Am I missing something?

**Comment by rivenirvana** [comment, marked as answer]
You're on Wayland, I'm assuming? See `restart-shell` in Makefile. `xprop -root &> /dev/null` returns true for me in Fedora GNOME even while using Wayland so it kills gnome-shell expecting it to restart like it does for X11. I used to use `make local-install` as well, but since moving to Wayland, I just do the routines `local-install` does except for `restart-shell`, like so: `make depcheck compile install configure enable`.

I do wonder if using something like `$XDG_SESSION_TYPE` is more reliable than the current condition being used.

**Reply by huyz** [reply]
Yes I'm on Wayland. Thanks

**Reply by ibrahima** [reply]
Wow thanks, I ran into the same issue as well on Ubuntu Jammy, and your workaround worked for me! I wonder if the README should be updated until the issue is resolved.

**Reply by huyz** [reply]
The README mentions it but in the wrong section (for Developers):

> While working on the shell, you can recompile, reconfigure, reinstall, and restart GNOME Shell with logging with make debug. Note that this only works reliably in X11 sessions, since Wayland will exit to the login screen on restarting the shell.
